content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -553,6 +581,37 @@ This admission controller also protects the access to `metadata.ownerReferences[
 of an object, so that only users with "update" permission to the `finalizers`
 subresource of the referenced *owner* can change it.
 
+### PersistentVolumeClaimResize {#persistentvolumeclaimresize}
+
```

Comment:
nit: I know this is mostly a move, but I'd expect to see something like:
```markdown
{{< feature-state for_k8s_version="v1.XX" state="alpha" >}}
```
here.